extensions/resource-deployment/src/ui/deployAzureSQLDBWizard/pages/databaseSettingsPage.ts

```diff
@@ -0,0 +1,306 @@
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import { DeployAzureSQLDBWizard } from '../deployAzureSQLDBWizard';
+import * as constants from '../constants';
+import { BasePage } from './basePage';
+
+export class DatabaseSettingsPage extends BasePage {
+
+	// ip address components
+	// private _newPublicIpCheckbox!: azdata.CheckBoxComponent;
+	// private _publicIpFlexContainer !: azdata.FlexContainer;
+	// private _publicIpDropdown!: azdata.DropDownComponent;
+	// private _publicIpDropdownLoader!: azdata.LoadingComponent;
+	// private _publicIpNetworkText!: azdata.InputBoxComponent;
+	private _startIpAddressTextRow!: azdata.FlexContainer;
+	private _startIpAddressTextbox!: azdata.InputBoxComponent;
+	private _endIpAddressTextRow!: azdata.FlexContainer;
+	private _endIpAddressTextbox!: azdata.InputBoxComponent;
+	private _firewallRuleNameTextbox!: azdata.InputBoxComponent;
+	private _firewallRuleNameTextRow!: azdata.FlexContainer;
+	private _databaseNameTextbox!: azdata.InputBoxComponent;
+	private _databaseNameTextRow!: azdata.FlexContainer;
+
+	private _form!: azdata.FormContainer;
+
+	constructor(wizard: DeployAzureSQLDBWizard) {
+		super(
+			constants.DatabaseSettingsPageTitle,
+			constants.DatabaseSettingsPageDescription,
+			wizard
+		);
+	}
+
+	public async initialize() {
+		this.pageObject.registerContent(async (view: azdata.ModelView) => {
+			await Promise.all([
+				//this.createPublicIPDropdown(view),
+				this.createIpAddressText(view),
+				this.createFirewallNameText(view),
+				this.createDatabaseNameText(view)
+			]);
+			this._form = view.modelBuilder.formContainer()
+				.withFormItems(
+					[
+						// {
+						// 	component: this.wizard.createFormRowComponent(view, constants.PublicIPDropdownLabel, '', this._publicIpFlexContainer, true)
+						// },
+						{
+							component: this._startIpAddressTextRow
+						},
+						{
+							component: this._endIpAddressTextRow
+						},
+						{
+							component: this._firewallRuleNameTextRow
+						},
+						{
+							component: this._databaseNameTextRow
+						}
+					],
+					{
+						horizontal: false,
+						componentWidth: '100%'
+					})
+				.withLayout({ width: '100%' })
+				.component();
+
+			return view.initializeModel(this._form);
+		});
+	}
+
+	public async onEnter(): Promise<void> {
+		//this.populatePublicIpkDropdown();
+		this.liveValidation = false;
+		this.wizard.wizardObject.registerNavigationValidator((pcInfo) => {
+			if (pcInfo.newPage < pcInfo.lastPage) {
+				return true;
+			}
+			this.liveValidation = true;
+			let errorMessage = this.formValidation();
+
+			if (errorMessage !== '') {
+				return false;
+			}
+			return true;
+		});
+	}
+
+	public onLeave(): void {
+		this.wizard.wizardObject.registerNavigationValidator((pcInfo) => {
+			return true;
+		});
+	}
+
+	// private async createPublicIPDropdown(view: azdata.ModelView) {
```

Comment:
lots of dead code in this PR.  If we want to commit this code like this, it's good practice to add a comment why the code is disabled and a name\date of who's adding the deadcode.  that helps when trying to clean-up these comment blocks in the future.